examples/src/main/java/jaxrs/examples/bootstrap/ClientAuthenticationJavaSeBootstrapExample.java

```diff
@@ -34,19 +34,15 @@
  * (e. g. 443, 8443, or someting completely different). The actual configuration needs to be queried after
  * bootstrapping, otherwise callers would be unaware of the actual chosen port. If the client's certificate is invalid
  * or cannot be validated, the server will reject the connection.
- * </p>
```

Comment:
I'm no HTML guru, but why is it wrong to use a closing tag here?

Reply:
Javadoc isn't pure HTML, it